src/docs/patterns/appbar.mdx

```diff
@@ -1,43 +1,86 @@
 
-<!-- KEYWORDS: design pattern appbar search collapse -->
+<!-- KEYWORDS: design pattern appbar search collapse toolbar banner -->
 
-# App Bar Patterns
+# App Bars
 
-App bars are useful in a variety of locations throughout PX Blue applications. They are typically used to provide a title or description for a collection of data that follows. They may also contain a menu and/or various actions that can be performed.
+<ImageGrid images={[Banner]} regularWidth />
 
-<MaterialDesignDescription title={'App Bars: Top'} url={'https://material.io/components/app-bars-top/'} />
+App bars, sometimes called toolbars, are often one of the first elements you see in an application. They are primarily used to provide context for where a user is in the hierarchy of the application.
 
-## A Basic App Bar
+<React.Fragment>
+    <MaterialDesignDescription title={'App Bars: Top'} url={'https://material.io/components/app-bars-top/'} />
+    <Divider />
+</React.Fragment>
 
-The most basic app bar is used at the top of a page as a title bar. It will have a menu button to the left (used to toggle a navigation menu / drawer) followed by one or two lines of text (title / subtitle) and, optionally, a dropdown with global page actions. Basic app bars can also be used to provide more detailed headers for Cards.
+# Basic App Bar
 
-## Collapsible App Bar
+The basic app bar provides information about the current screen. Typically, this is a short title that indicates which screen the user is on (e.g., "Settings"). However, the app bar can also contain a navigation icon, subtitle, and/or various global action buttons for the screen. When placing actions in the app bar, limit yourself to no more than three. If you have more actions than this, consider grouping them into a menu.
+
+<ImageGrid regularWidth images={[BasicBars]} caption={'Basic app bars in order of increasing complexity.'} />
+
+## Dropdown Toolbar
+
+A special case of the basic app bar is the dropdown toolbar. This version transforms the simple text subtitle of the app bar into a dropdown menu that can be used to configure the page. For example, you may have a dropdown that allows users to switch between different physical locations on an overview screen.
+
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[DropdownVariations]}
+        caption={
+            ('1. On mobile, the dropdown toolbar opens the options panel in a bottom sheet.',
+            '2. On desktop, the dropdown opens as a menu below the toolbar.')
+        }
+    />
+    <DemoCard
+        repository={'Dropdown Toolbar'}
+        angular={
+            'https://pxblue-components.github.io/angular/?path=/story/components-dropdown-toolbar--with-full-config'
+        }
+        react={'https://pxblue-components.github.io/react/?path=/story/components-dropdown-toolbar--with-full-config'}
+        ionic
+        reactNative
+    />
+    <Divider />
+</React.Fragment>
```

Comment:
I think all the other patterns are following the image-demo-text order? Correct me if I'm wrong

Reply:
As far as I can tell, there's no consistency.

I see on different pages:
`image-text-demo-image`
`image-text-demo`
`image-demo-text`
`text-image-demo`
`text-demo`
and
`demo-text`

I'm kind of preferential to text first to give some context around the images/demos.

Reply:
the ones currently doing image-demo-text are all ones that haven't been overhauled yet